test/components/views/settings/CrossSigningPanel-test.tsx

```diff
@@ -38,7 +38,15 @@ describe("<CrossSigningPanel />", () => {
     beforeEach(() => {
         mockClient.doesServerSupportUnstableFeature.mockResolvedValue(true);
         mockClient.isCrossSigningReady.mockResolvedValue(false);
-        mocked(mockClient.crypto!.crossSigningInfo).isStoredInSecretStorage.mockClear().mockResolvedValue(null);
+        mocked(mockClient.getCrypto()!.getCrossSigningStatus).mockResolvedValue({
+            publicKeysOnDevice: true,
+            privateKeysInSecretStorage: false,
+            privateKeysCachedLocally: {
+                masterKey: true,
+                selfSigningKey: true,
+                userSigningKey: true,
+            },
+        });
```

Comment:
isn't this the default, from `mockClientMethodsCrypto` ?

Reply:
Yes but I'm overriding it in some tests. So in order to not disturb the other tests, I'm setting the default value here before each test

Reply:
bleh; I find this quite confusing.

Can we move the setup of `mockClient` into `beforeEach`?

Reply:
Yes done :)